Automattic/wp-calypso
# Plugins Browser: exclude specific plugin from plugin view

#### Changes proposed in this Pull Request

﻿Let's allow excluding a specific plugin from the plugins view when we do not want to promote it

#### Testing instructions

* Spin up this branch
* Go to `http://calypso.localhost:3000/plugins/yoursite.com`
* Scroll down to the "new" section
    * You should not see any plugin using the `zamir` slug.
